parts/linux/cloud-init/artifacts/cgroup-memory-telemetry.sh

```diff
@@ -80,5 +80,5 @@ EVENT_JSON=$( jq -n \
     --arg EventTid      "0" \
     '{Timestamp: $Timestamp, OperationId: $OperationId, Version: $Version, TaskName: $TaskName, EventLevel: $EventLevel, Message: $Message, EventPid: $EventPid, EventTid: $EventTid}'
 )
-
+mkdir -p ${EVENTS_LOGGING_DIR}
```

Comment:
we shouldn't need to do this, WALinuxAgent will do this before us before cloud-init/CSE is invoked